pages/common/pax.md

```diff
@@ -0,0 +1,27 @@
+# pax
+
+> Archiving and copying utility.
+
+- List the contents of an archive:
+
+`pax -f {{archive.tar}}`
+
+- List the contents of a gzipped archive:
+
+`pax -zf {{archive.tar.gz}}`
+
+- Create an archive from files:
+
+`pax -wf {{target.tar}} {{file1 file2 file3}}`
```

Comment:
Perhaps each file could be enclosed in its own set of `{{` and `}}`, and be prepended with `path/to/` please? The same goes for other examples here.

Reply:
I had oriented myself on `tar.md`, but sure I can do that. The changes are pushed now.